src/reference/modules/administration/pages/security.adoc

```diff
@@ -34,6 +34,13 @@ rudder.server.hstsIncludeSubDomains=true
 rudder.auth.idle-timeout=15 minutes
 ----
 
+* When a user have not logged in, by default for 90 days, the account is marked as *disabled*, you can change this value by modifying or adding the parameter `rudder.users.cleanup.account.disableAfterLastLogin` in `/opt/rudder/etc/rudder-web.properties` by following this format : https://scala-lang.org/api/3.x/scala/concurrent/duration/Duration$.html#apply-537
+
+* A user can be *deleted* after a certain period of time, by default the parameter `rudder.users.cleanup.account.deletedAfterLastLogin` is set to 120 days. You can change this value by modifying or adding this parameter in `/opt/rudder/etc/rudder-web.properties` by following this format : https://scala-lang.org/api/3.x/scala/concurrent/duration/Duration$.html#apply-537
```

Comment:
```suggestion
* A user is *deleted* after a certain period of time, by default the parameter `rudder.users.cleanup.account.deletedAfterLastLogin` is set to 120 days. You can change this value by modifying or adding this parameter in `/opt/rudder/etc/rudder-web.properties` by following this format : https://scala-lang.org/api/3.x/scala/concurrent/duration/Duration$.html#apply-537
```

Reply:
I don't think the user wants to open the scaladoc, we can hardcode the format, and users will not likely want anything else than days, that leaves : `d, day`